stdlib/Profile/test/allocs.jl

```diff
@@ -73,8 +73,12 @@ end
     @test length(first_alloc.stacktrace) > 0
     @test length(string(first_alloc.type)) > 0
 
-    @testset for type in (Task, Vector{Float64},)
-        @test length(filter(a->a.type <: type, profile.allocs)) >= NUM_TASKS
+    # This test does not work with MMTk because of fastpath allocation
+    # which never calls the allocation profiler.
```

Comment:
This is fine for now, but we will want to port these observability tools (e.g. allocation profiler and heap snapshot) to MMTk in the future.

Could you add a `TODO` here?